src/MoviesList/MoviesList.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import MovieCard from 'src/MovieCard';
```

Comment:
seems this component should be placed in `MovieList` folder as it's used only there, what do you think?

Reply:
I agree and implemeted the same